src/Cache/MySQL.php

```diff
@@ -365,7 +368,8 @@ public function load()
 					{
 						while ($row = $query->fetchColumn())
 						{
-							$feed['child'][\SimplePie\SimplePie::NAMESPACE_ATOM_10]['entry'][] = unserialize($row);
+							assert(is_string($row));
```

Comment:
The assertions are only used here as a hint for the static analyser – they should never trigger, unless the database is corrupted. But I do not really like them.

Reply:
Would that help the static analyser?

```diff
- assert(is_string($row));
+ $row = (string) $row;
```

Reply:
Yes, that would make the type checker happy I assume. Thought I find this somewhat iffy too, since it casts a variable to string that already is a string (unless db is corrupted). Assertions, meanwhile, will be disabled on production.

I know this will have negligible performance impact but it is still a smelly code. Though maybe it is acceptable trade off, if we want static analysis to be happy.